[*] Running ...
[*] Dataset loaded: E:\Projects\CO\reevo-main/problems/mkp_aco\dataset/train100_dataset.npz with 5 instances.
Traceback (most recent call last):
  File "E:\Projects\CO\reevo-main/problems/mkp_aco/eval.py", line 58, in <module>
    obj = solve(prize, weight)
          ^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main/problems/mkp_aco/eval.py", line 23, in solve
    heu = heuristics(prize.copy(), weight.copy()) + 1e-9
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main\problems\mkp_aco\gpt.py", line 71, in heuristics_v2
    # Update heuristics with refined scores
                           ^^^^^^^^^^^^^^^^^
IndexError: invalid index to scalar variable.
